colossus-metrics/src/main/scala/colossus/metrics/Collection.scala

```diff
@@ -146,3 +141,9 @@ class Collection(val config: CollectorConfig) {
   }
 
 }
+
+object Collection{
+  def withReferenceConf(intervals : Seq[FiniteDuration]) : Collection = {
```

Comment:
It looks like this is only used in tests?  Might be a good idea either to add a comment about what it's used for, or just move it into tests.